### PR TITLE
adding files for Bigfoot Game

### LINKED ISSUE #66: Find Bigfoot using HTML, CSS & Javascript
I would like to add a game to find Bigfoot using HTML, CSS & Javascript to this repo.

### PR DESCRIPTION
**Pull request by iqtheengineer**
close #66 
 - Created a separate branch
- Created project folder inside /public/Projects
- HTML file named index.html
- Gif uploaded to assets folder (bigfoot.gif).
- Project folder has an info.txt file with required fields.
@SandeepKrSuman

### CODE REVIEW

File: public/Projects/IQtheEngineerBigfootGame/info.txt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+    name: "Iman Crooks",
+    ghurl: "https://github.com/iqtheengineer",
+    imgName: "assets/bigfoot.gif",
```
Comment (SandeepKrSuman) [inline public/Projects/IQtheEngineerBigfootGame/info.txt:4]: put the gif inside `public/assets` folder. (All the gifs of different projects are present there)

the imgName should be the gif name only and not the path to the gif. 

For example,

```
imgName: "bigfoot.gif"
```

**Review by SandeepKrSuman — CHANGES_REQUESTED** [review]
Also, kindly add some instruction about the game for new users. 
You can add steps to play the game before starting the game.

File: public/Projects/IQtheEngineerBigfootGame/info.txt (hunk 1 above)
Comment (iqtheengineer) [reply public/Projects/IQtheEngineerBigfootGame/info.txt:4]: Added commit (a2490ca8f011f008a4acf76996e33220ad5fc7d8) to add game instructions via a prompt message and relocated gif to public/assets folder and renamed to bigfoot.gif. Please let me know if anything else is needed. @SandeepKrSuman

Comment (SandeepKrSuman) [reply public/Projects/IQtheEngineerBigfootGame/info.txt:4]: Just modify info.txt file

remove ----> `imgName: "assets/bigfoot.gif"`

add ----->  `imgName: "bigfoot.gif"`

Just do this. Everything else is fine.

**Review by SandeepKrSuman — CHANGES_REQUESTED** [review]

**Comment by iqtheengineer** [comment]
My apologies, not sure why that file wasn't included in the previous commit. I've made the correction. @SandeepKrSuman

**Review by SandeepKrSuman — APPROVED** [review]
Looks good 🚀

**Pull request merged**